### PR TITLE
Add uw-uploadfirst

### PR DESCRIPTION
**Pull request by DannyS712**
Another single-issue notice template. Solves #489. This is my first time trying this, so please tell me if I did something wrong.

### CODE REVIEW

File: modules/twinklewarn.js [hunk 1]
```diff
@@ -1015,6 +1015,10 @@ Twinkle.warn.messages = {
 			label: "Posting at the top of talk pages",
 			summary: "Notice: Posting at the top of talk pages"
 		},
+		"uw-uploadfirst": {
+			label: "Uploading images",
+			summary: "Notice: Uploading images"
+		}
```
Comment (JJMC89) [inline modules/twinklewarn.js]:
```suggestion
		},
```
Missing comma

**Comment by Amorymeltzer** [comment]
Thanks for the contribution @DannyS712!  This was apparently part of Twinkle but was removed in 2015 as part of #302 by @MusikAnimal.  Any thoughts MA?

**Comment by DannyS712** [comment]
@Amorymeltzer 2015 isn't the last time this was brought up - @MusikAnimal wrote 9 months ago in #344 that it wasn't particularly needed. Maybe it shouldn't be added back?

**Comment by Amorymeltzer** [comment]
Good find, thanks for the follow-up!  Closing per the rationale in #344.

**Pull request closed without merging**